en/docs/guides/login/oidc-discovery.md

```diff
@@ -267,7 +267,6 @@ Follow the steps below to obtain configuration details of the OpenID Provider.
 -----
 
 !!! info "Related Topics"
-<<<<<<< HEAD
     - [Concept: OpenID Connect Discovery](../../../references/concepts/authentication/discovery)
     - [Guide: OpenID Connect Grant Types](../../access-delegation/oidc-grant-types)
```

Comment:
Does this link work? 

If not please change it to (../../../guides/access-delegation/oidc-grant-types)

Reply:
yes it works